plugins/scaffolder-react/report-alpha.api.md

```diff
@@ -69,6 +70,35 @@ export function createFormField<
   TUiOptions extends z.ZodType,
 >(opts: FormFieldExtensionData<TReturnValue, TUiOptions>): FormField;
 
+// @public
+export function createScaffolderFormDecorator<
```

Comment:
Does it make sense to expose it to zod as it is done here? https://github.com/backstage/backstage/pull/26957

Reply:
I think that we want to align so that zod is exposed like the below:

https://github.com/backstage/backstage/pull/25975/files#diff-133af4d9fd32f76d048c15eee319683f33ae9b8045d4a258a84923c03f9ec892R78-R80

It's copying how the new frontend system exposes zod, so that you write things like this:

```ts
createScaffolderFormDecorator({
  input: {
    myInput: (z) => z.string().optional(),
  }
});
```

Same as this https://backstage.io/docs/frontend-system/architecture/extensions#extension-configuration

I think that other PR needs updating, just haven't had chance to leave feedback there yet.